### ISSUE TITLE
libswiftAVFoundation.dylib image not found

### ISSUE THREAD

**Issue by dgobaud** [onfido/onfido-ios-sdk#93]
### What was the expected behaviour?

app runs

### What happened instead?

app crashes

dyld: Library not loaded: @rpath/libswiftAVFoundation.dylib
  Referenced from: /Users/davidgobaud/Library/Developer/CoreSimulator/Devices/0407934D-1CCA-4EEA-BC1B-CE13FC7D384D/data/Containers/Bundle/Application/2FEC8BC4-DE6C-4E86-A4D4-1ED5DE8930F6/passfolio-debug.app/Frameworks/Onfido.framework/Onfido
  Reason: image not found

### Version info:

- Cocoapods (if applicable):
- Onfido SDK (Debug/Release):
- iOS: 12.2
- Xcode: 10.2.1
- Device/Simulator: iPhone X
- Device/Simulator language: Objective-C + React Native

### Steps to reproduce:

run app

**Comment by dgobaud** [comment]
"Always Embed Swift Standard Libraries" in XCode build settings is set to Yes

**Comment by cpetzel** [comment]
This also happens to me after updating to xcode 10.2 and SDK 11.1.1.

**Comment by anuragajwani** [comment]
Hi all,

I have created a new project using Xcode 10.2.1. I have installed Onfido SDK 11.1.1 via Cocoapods and ran it on a device running 12.2. It runs successfully.

Can anyone share a project where this is reproducible please?
[GithubIssue93.zip](https://github.com//onfido-ios-sdk/files/3208350/GithubIssue93.zip)

Anurag

**Comment by dgobaud** [comment]
@anuragajwani in our podfile we have below is this correct config? maybe we have outdated version?

```
  pod 'Onfido', :configurations => ['Debug']
  pod 'Onfido-Release', :configurations => ['Release']
```

**Comment by dgobaud** [comment]
actually i see in lock file below. maybe we have outdated version will try updating

```
  - Onfido (10.4.0)
  - Onfido-Release (10.4.0)
```

**Comment by vnvs** [comment]
I updated and this error is happening now:

<img width="509" alt="Captura de Tela 2019-05-22 às 15 32 21" src="https://user-images.githubusercontent.com/4933389/58199573-272aa680-7ca7-11e9-8385-49b15771bcb2.png">

**Comment by dgobaud** [comment]
@anuragajwani it is working with new XCode! I think @vnvs needs to update but i think we can close this. basically the answer is update to latest OnFido pod

**Comment by dgobaud** [comment]
@anuragajwani another dev is having hte problem in iOS 12.1 simualtor below error - does it mean the OnFido SDK will not work in iOS 12.1? any idea how to fix it? thanks

dyld: Library not loaded: @rpath/libswiftAVFoundation.dylib
Referenced from: /Users/victorfernandez/Library/Developer/CoreSimulator/Devices/6E0A478D-9074-4E89-B3E5-8D7E217C6045/data/Containers/Bundle/Application/4598020B-AB2F-4787-8A31-152B32E37E78/passfolio.app/Frameworks/Onfido.framework/Onfido
Reason: no suitable image found. Did find:
/usr/lib/swift/libswiftAVFoundation.dylib: mach-o, but not built for iOS simulator
(lldb)

**Comment by anuragajwani** [comment]
@dgobaud I am still unable to reproduce. I can't say for certain why this is happening. But I have a few guesses.

[Here](https://developer.apple.com/documentation/xcode_release_notes/xcode_10_2_1_release_notes) are the release notes for Xcode 10.2.1:
```
Simulator
Known Issues
Simulators for iOS 9.3 and earlier might fail to launch Swift apps with the message: “dyld: Library not loaded: /usr/lib/libauto.dylib”. (49326587)
Workaround: Run the following command in Terminal for the relevant version of iOS:
sudo mkdir '/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS 9.3.simruntime/Contents/Resources/RuntimeRoot/usr/lib/swift'
```

It looks like Xcode is not loading the appropriate libraries in some cases. I experienced this issue briefly when Xcode 10.2.0 was released and it only happened on iOS 12.2. Since iOS 12.2 Swift libraries should be included in the operating system. Prior Swift libraries had to be packaged with the app itself. This change can have an influence in the issue you are expereincing.

As I can't reproduce this error consistently I can't help much. However if you can provide me a project that can consitently reproduce this issue it would be of great help to get to a solution.

Anurag

**Comment by dgobaud** [comment]
Ok guess it is an Xcode bug so I'll close this thanks for the help
